### ISSUE TITLE
`preventOverlayInteraction` doesn't remove all blocking listeners

### ISSUE THREAD

**Issue by spencerjsmall** [fatihsolhan/v-onboarding#106]
Hi! It appears that even with `preventOverlayInteraction` set to false, this package still applies blocking dom listeners that prevent clicks, focusin, etc.. The following workaround works locally, however in my production app these event listeners get wrapped by sentry and I'm unable to intercept them the same:
```
 const originalAddEventListener = document.addEventListener;
    document.addEventListener = function (type, listener, options) {
      const listenerStr = listener.toString();
      if (
        (type === 'click' && listenerStr.includes('checkClick')) ||
        (type === 'touchstart' && listenerStr.includes('checkPointerDown')) ||
        (type === 'mousedown' && listenerStr.includes('checkPointerDown')) ||
        (type === 'keydown' && listenerStr.includes('checkKey')) ||
        (type === 'focusin' && listenerStr.includes('checkFocusIn'))
      ) {
        return;
      }
      originalAddEventListener.call(this, type, listener, options);
    };
```
Is there a reason these blocking dom listeners are still being added? 
Thank you!

**Comment by spencerjsmall** [comment]
I see now that this is from using `focusTrap` in the step component. In my opinion, this should be turned off if `preventOverlayInteraction` is false.

**Comment by magicyoda** [comment]
Same issue here, when trying to close or show an element on the GUI for the next step, I try to simulate a click programatically but it fails as the events are prevented on the overlay. This even when setting 'preventOverlayInteraction' to false.

**Comment by fatihsolhan** [comment]
Hi everyone! 👋

This issue has been resolved and included in v2.9.0.

Feel free to reopen if you encounter any issues. Thanks for your patience! 🙌
